modules/modal/main/index.coffee

```diff
@@ -24,21 +24,19 @@ class Modal extends hx.EventEmitter
 
 
   show: (cb) ->
+    # Clears the focus so pressing 'enter' does not cause buttons to call modal.show()
+    document.activeElement.blur()
```

Comment:
Can document.activeElement be null, eg if the modal was shown programmatically ?